jaeger-spark-dependencies-elasticsearch/src/main/java/io/jaegertracing/spark/dependencies/elastic/ElasticsearchDependenciesJob.java

```diff
@@ -209,7 +215,27 @@ private static String prefix(String prefix) {
   }
 
   public void run(String peerServiceTag) {
-    run(indexDate("jaeger-span"), indexDate("jaeger-dependencies") ,peerServiceTag);
+
+    String[] readIndices;
+    String[] writeIndex;
+
+    // read spans from alias index name
+    if (this.useReadAlias) {
```

Comment:
The indentation looks inconsistent from here down.

Reply:
fixed.